src/pipecat/audio/filters/krisp_filter.py

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright (c) 2024, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+import numpy as np
+import os
+
+from pipecat.audio.filters.base_audio_filter import BaseAudioFilter
+from loguru import logger
+from pipecat.frames.frames import FilterControlFrame, FilterEnableFrame
+
+try:
+    from pipecat_krisp import KrispAudioProcessor
+except ModuleNotFoundError as e:
+    logger.error(f"Exception: {e}")
+    logger.error(
+        "In order to use the Krisp filter, you need to download and install `pipecat-krisp`:"
+        "\n https://github.com/daily-co/pipecat-krisp/."
+        "\n `pip install --editable ../pipecat-krisp`."
+    )
```

Comment:
I think we should publish `pipecat-ai-krisp` to pypi and make releases, etc. So people just have to do `pip install pipecat-ai[krisp]`